packages/node/src/lib/process-request.ts

```diff
@@ -207,14 +210,16 @@ export default function processRequest(
     queryString: searchToArray(reqUrl.searchParams),
     postData,
     // TODO: When readme starts accepting these, send the correct values
-    cookies: [],
+    cookies: [] as Cookie[],
     headersSize: -1,
     bodySize: -1,
   };
```

Comment:
if you add `as const` to the end of object definitions like this you get a lot of nice additional typing features!

Reply:
love it! https://github.com/readmeio/metrics-sdks/pull/900/commits/1585f05c5877bf90e6d5f9c920cb160a2aaf3629